Deleted Tracking Plan (#6867)
segmentio/segment-docs: src/protocols/faq.md

### Can I import events from my Source Schema into a Tracking Plan?

When you initially create your Tracking Plan, you can import events into it from a Source Schema. Manually add these events by clicking the the (+) next to the event in your Source Schema page after connecting your Tracking Plan.

### Can I recover a Tracking Plan that was deleted?

> At this time it is not possible to recover a deleted Tracking Plan. Please delete with caution. 

Reviewer: ```suggestion
Segment is unable to recover a deleted Tracking Plan. 
```
